packages/date-utils/src/types/locales.ts

```diff
@@ -0,0 +1,7 @@
+export const SupportedLocales = {
```

Comment:
Why are these the only supported ones?

Reply:
That's what Sandy initially specced https://www.figma.com/design/4h2VwjCuJUbeZ7hzD2J1rq/LeafyGreen-Design-System?m=auto&node-id=21841-205738&t=RTp2r9ajPKtlNLk8-1

Reply:
We test some additional ones, but these are the only ones we "officially" support 
https://github.com/mongodb/leafygreen-ui/blob/main/packages/date-utils/src/testing/testValues.ts